### ISSUE TITLE
sqlite migration:generate not work :(

### ISSUE THREAD

**Issue by EndyKaufman** [typeorm/typeorm#1380]
.env
```bash
DATABASE_URL=sqlite://db.sqlite3
```
ormconfig.js
```js
const ConnectionString = require('connection-string').ConnectionString;
const config = require('dotenv').config;
const path = require('path');

config();

const connectionString = new ConnectionString(process.env.DATABASE_URL);

if (connectionString.protocol === 'sqlite') {
    module.exports = {
        type: 'sqlite',
        database: connectionString.host,
        entities: [
            __dirname + '/src/libs/**/*.entity.ts',
        ],
        migrations: [
            'src/migrations/**/*.ts'
        ],
        subscribers: [
            'src/subscribers/**/*.ts'
        ],
        cli: {
            'migrationsDir': 'src/migrations',
            'subscribersDir': 'src/subscribers'
        },
        logging: 'all',
        synchronize: false,
    }
} else {
    module.exports = {
        type: connectionString.protocol,
        host: connectionString.host,
        port: +connectionString.port,
        username: connectionString.user,
        password: connectionString.password,
        database: connectionString.segments[0],
        entities: [
            __dirname + '/src/libs/**/*.entity.ts',
        ],
        migrations: [
            'src/migrations/**/*.ts'
        ],
        subscribers: [
            'src/subscribers/**/*.ts'
        ],
        cli: {
            'migrationsDir': 'src/migrations',
            'subscribersDir': 'src/subscribers'
        },
        logging: 'all',
        synchronize: true,
    }
}
```
not work command
```bash
ts-node ./node_modules/typeorm/cli.js migrations:generate -n init
```

**Comment by pleerock** [comment]
Do you have some error or what issue do you have? Also are you using typescript or not?

**Comment by EndyKaufman** [comment]
just nothing happens, black screen
yes entities write on typescript

**Comment by EndyKaufman** [comment]
if write console.log in 
C:\Projects\typeorm_test\node_modules\typeorm\driver\sqlite\SqliteDriver.js#L61
```js
    function SqliteDriver(connection) {
        var _this = _super.call(this, connection) || this;
        _this.connection = connection;
        _this.options = connection.options;
        _this.database = _this.options.database;
        console.log(_this.options);
        // validate options to make sure everything is set
        if (!_this.options.database)
            throw new DriverOptionNotSetError_1.DriverOptionNotSetError("database");
        // load sqlite package
        _this.loadDependencies();
        return _this;
    }
```
output path is wrong
```
ts-node ./node_modules/typeorm/cli.js migrations:generate "-n" "init"

{ type: 'sqlite',
  database: 'C:\\Projects\\typeorm_test/C:\\Projects\\typeorm_test/db.sqlite3',
  entities: [ 'C:\\Projects\\typeorm_test/src/libs/**/*.entity.ts' ],
  migrations: [ 'src/migrations/**/*.ts' ],
  subscribers: [ 'src/subscribers/**/*.ts' ],
  cli:
   { migrationsDir: 'src/migrations',
     subscribersDir: 'src/subscribers' },
  logging: false,
  synchronize: false,
  migrationsRun: false,
  dropSchema: false }
```

**Comment by pleerock** [comment]
are you able to setup a git repo with reproduction of the problem?

**Comment by EndyKaufman** [comment]
repository: https://github.com/EndyKaufman/typeorm_issues_1380
run: ```npm test```

**Comment by pleerock** [comment]
Looks like you are using `0.1.6`. Can you please upgrade your typeorm version to the latest, `0.1.9` and check again?

**Comment by pleerock** [comment]
ah, no, sorry looks like you are using `next`. Are you able to check it in 0.1.9? Because we already had such issue and it should be fixed in 0.1.9, maybe those changes just wasn't merged in next

**Comment by EndyKaufman** [comment]
I upgraded to the next version, on all the previous ones did not work, just checked once again for 0.1.9 - does not work :(

still found bugs from one to many, now I will write the issues)

**Comment by EndyKaufman** [comment]
It's a bug, or I'm doing something wrong
just want to use migrations in the sqlite

**Comment by pleerock** [comment]
ah wait. Sorry, I thought you were talking about `migrations:create` command. Migration generation is only supported in mysql and partially in postgres currently. We are planning to implement migration generation in sqlite in `0.2.0`. For now you'll need to use `synchronize` option or write migrations manually.

**Comment by AlexMesser** [comment]
fixed and released in `typeorm@0.2.0-alpha.23`. You can try it via `npm i typeorm@next`.

**Comment by kunalgrover05** [comment]
I tried using migrations with a sqlite connection with TypeORM@next but it says No changes in database schema were found - cannot generate a migration. To create a new empty migration use "typeorm migrations:create" command

I used an example similar to the above linked by Endy and I get this on generating a migration: "No changes in database schema were found - cannot generate a migration. To create a new empty migration use "typeorm migrations:create" command"
